src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -546,20 +546,20 @@ private boolean isWrongOrder(String name, boolean isStatic) {
         final boolean result;
         if (isStatic) {
             if (useContainerOrderingForStatic) {
-                result = compareContainerOrder(lastImport, name, caseSensitive) > 0;
+                result = compareContainerOrder(lastImport, name, caseSensitive) >= 0;
             }
             else if (option == ImportOrderOption.TOP
                 || option == ImportOrderOption.BOTTOM) {
                 result = sortStaticImportsAlphabetically
-                    && compare(lastImport, name, caseSensitive) > 0;
+                    && compare(lastImport, name, caseSensitive) >= 0;
             }
             else {
-                result = compare(lastImport, name, caseSensitive) > 0;
+                result = compare(lastImport, name, caseSensitive) >= 0;
             }
         }
         else {
             // out of lexicographic order
-            result = compare(lastImport, name, caseSensitive) > 0;
+            result = compare(lastImport, name, caseSensitive) >= 0;
```

Comment:
Should we throw other warning message for repetitive imports? Or just import ordering message.

Reply:
report only on import ordering.
Fact that import is duplicated is not this check business.

Reply:
> Fact that import is duplicated is not this check business.

Right, maybe we can add new feature to detect duplicated import in the future.But, I can only cover the red lines by this method now.

Reply:
in this PR please focus on passing CIs